### DISCUSSION TITLE
How to use Titiler with openlayers?

### DISCUSSION THREAD

**Discussion by vijay2712** [developmentseed/titiler#434] [Q&A]
Please give any documentation to use Titiler with Openlayers

**Comment by vincentsarago** [comment]
Please ask precise question!

**Comment by vijay2712** [comment]
I am not able to use Titiler with openlayers

**Comment by vincentsarago** [comment]
I'm not able to answer vague question

**Reply by vijay2712** [reply]
It's not a vague question.  I want to use your Titiler in my Software, Is it a vaque question?

**Reply by vijay2712** [reply]
If you have developed some software then you should answers the queries properly,

**Reply by vijay2712** [reply]
Are you the owner of this repo?

**Reply by vincentsarago** [reply]
Dear @vijay2712 I'm sorry but I'll lock this discussion because clearly you don't want to explain your issue or didn't read the documentation. 

- I'm not sure what you mean but `use TiTiler with OpenLayers`.
- TiTiler is a python library that will help you creating dynamic raster tile services (e.g WMTS). It is agnostic to the client library (e.g mapbox/leaflet/openlayer) so please checkout OpenLayers documentation to see how it works
- Maybe English is not your first language but please don't start your discussion by `please do/give/fix ...`

> If you have developed some software then you should answers the queries properly,

what is the query? 

> Are you the owner of this repo?

do you know how GitHub works? https://github.com/developmentseed/titiler/graphs/contributors
does it change something if I'm not the owner. I can't help you if the question is not specific 🤷‍♂️
